packages/beacon-node/src/chain/chain.ts

```diff
@@ -1,4 +1,6 @@
 import path from "node:path";
+import fs from "node:fs";
+import {promisify} from "node:util";
```

Comment:
it is better to just import the promise-based functions from `fs/promises`

```suggestion
import fs from "node:fs/promises";
```

Reply:
Thanks for the heads up. Just FYI I used `promisify` because it was used when writing invalid SSZ objects

https://github.com/ChainSafe/lodestar/blob/751969c03ae443032e9e4d062d1e15ff89a2d00a/packages/beacon-node/src/util/file.ts#L17

Reply:
yeah, not a big deal either way, but should avoid using `promisify` if native promise-based APIs exist